sql/catalyst/src/test/scala/org/apache/spark/sql/connector/catalog/InMemoryRowLevelOperationTable.scala

```diff
@@ -34,6 +34,8 @@ class InMemoryRowLevelOperationTable(
     properties: util.Map[String, String])
   extends InMemoryTable(name, schema, partitioning, properties) with SupportsRowLevelOperations {
 
+  var replacedPartitions: Seq[Seq[Any]] = Seq.empty
```

Comment:
Maybe add a comment to mention this is for test.

Reply:
Added a comment above.